www/caddy/src/opnsense/scripts/OPNsense/Caddy/caddy_diagnostics.py

```diff
@@ -101,6 +101,8 @@ async def extract_certificate_info(cert_path):
 
         # Extract the hostname from the filename
         hostname = os.path.basename(cert_path).replace('.crt', '').lower()
+        if hostname.startswith("wildcard_"):
+            hostname = hostname.replace("wildcard_", "*")
```

Comment:
```suggestion
            hostname = hostname.replace("wildcard_", "*", 1)
```

Reply:
but bonus points for trying to anchor it with the startswith :)

Reply:
Arent underscored illegal in domain names so this pattern should not occur anywhere else?

Reply:
why push your luck? :)

Reply:
I wont, I know that luck will not be on my side. This solution is janky anyway I should rather use direct CN extraction, but this patch is the simplest right now.